src/bin/flamegraph.rs

```diff
@@ -132,6 +132,14 @@ struct Opt {
     /// JavaScript in all of the test files.
     #[structopt(raw(hidden = "true"), long = "no-javascript")]
     no_javascript: bool,
+
+    /// Produce a flame chart (sort by time, do not merge stacks)
+    #[structopt(
+        long = "flamechart",
+        conflicts_with = "no-sort",
```

Comment:
I _think_ this should be `requires = "no-sort"`, unless I'm missing something?

Reply:
I do not remember why :). Right now I think differently.

Reply:
Oh, I think I see what's going on. Basically, flamechart mode _forcibly_ disables sorting (since it makes no sense with sorting enabled). So, either we should _require_ that the user also give `--no-sort`, or we should _reject_ it if the user passes `--no-sort` (since there's no need to). I'd lean towards the former, but either is fine.

Reply:
Indeed. Original code just ignore this option.